qiskit_ibm_runtime/session.py

```diff
@@ -25,6 +26,8 @@
 from .program.result_decoder import ResultDecoder
 from .ibm_backend import IBMBackend
 
+_session_setup_lock = Lock()
```

Comment:
why making this a global variable instead of part of `Session`? If I create 2 `Session` objects for 2 different backends, one shouldn't have to wait for the other.

Reply:
You're right. Actually, if you create two separated sessions, even for the same backend, they should not wait for each other. I will modify.